plugins/org.jkiss.dbeaver.ext.tidb.ui/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,24 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %Bundle-Name
+Bundle-SymbolicName: org.jkiss.dbeaver.ext.tidb.ui;singleton:=true
+Bundle-Version: 1.0.32.qualifier
+Bundle-Release-Date: 20221121
+Require-Bundle: org.eclipse.ui,
+ org.eclipse.core.runtime,
+ org.jkiss.dbeaver.model,
+ org.jkiss.dbeaver.model.sql,
+ org.jkiss.dbeaver.ext.generic,
+ org.jkiss.dbeaver.ext.mysql,
+ org.jkiss.dbeaver.ext.mysql.ui,
+ org.jkiss.dbeaver.ext.tidb,
+ org.jkiss.dbeaver.ext.oracle,
```

Comment:
You do not have Oracle bundle dependencies. What for it here?

Reply:
Just remove it from your code.

Reply:
Got it.